src/App.js

```diff
@@ -1,16 +1,158 @@
 import React from 'react';
+import './App.css';
+import TodoList from './components/TodoComponents/TodoList'
+import TodoForm from './components/TodoComponents/TodoForm';
+// import { InstantSearch, Hits } from 'react-instantsearch-dom';
 
 class App extends React.Component {
-  // you will need a place to store your state in this component.
-  // design `App` to be the parent component of your application.
-  // this component is going to take care of state, and any change handlers you need to work with your state
+  constructor() {
+    super();
+
+    this.state = { 
+      todos: [
+        {
+          task: 'Give the dog a bath',
+          id: 1,
+          completed: false,
+        }, {
+          task: 'Paint the walls',
+          id: 2,
+          completed: false,
+        }
+    ]
+  }
+}
+
+// Need a function to add a new todo object to the todo array
+addToDo = (newTask, id) => {
+  // Create a new todo object based on a given task and id
+  const todo = {
+    task: `${newTask}`,
+    id: id,
+    completed: false,
+  }
+
+  // Get the todo array and add the new todo to the end using push()
+  const todos = this.state.todos;
```

Comment:
this line should be `const todos = this.state.todos.slice()`, so you make a copy of the current todos in state, but don't actually modify it directly